--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -261,6 +261,7 @@
         PLATFORM_CPU_ARM64: [
             "include/cpuinfo_aarch64.h",
             "include/internal/cpuid_aarch64.h",
+            "include/internal/windows_utils.h",
         ],
         PLATFORM_CPU_MIPS: ["include/cpuinfo_mips.h"],
         PLATFORM_CPU_PPC: ["include/cpuinfo_ppc.h"],
@@ -324,6 +325,7 @@
         PLATFORM_CPU_ARM64: [
             "include/cpuinfo_aarch64.h",
             "include/internal/cpuid_aarch64.h",
+            "include/internal/windows_utils.h",
         ],
         PLATFORM_CPU_MIPS: ["include/cpuinfo_mips.h"],
         PLATFORM_CPU_PPC: ["include/cpuinfo_ppc.h"],
